### PR TITLE
implements #19

### PR DESCRIPTION
**Pull request by 73**
I guess this is ready to be merged now...

### CODE REVIEW

**Comment by humm** [comment]
Hum... seems this is not compliant with python 3---see the Travis CI report. As we have users using both versions of python, this is a really a requirement for any pull request.

**Comment by 73** [comment]
I see. Shouldn't be that hard to make it runnable with 2.7, 3.3-5 ... Do you have a tip on how to check this locally? I may just run PyCharm code inspection, but I'm not sure I will always have an IDE at hand.

**Comment by humm** [comment]
You just need to have an another version of python you can switch to for the tests. I personally use [pyenv](http:/github.com/yyuu/pyenv), but there are other ways, such as using `virtualenv`.

**Comment by 73** [comment]
Hey, 
so I used pyenv (virtualenv only provides my distros python versions) and went ahead and installed python 3.5.0 (which is exactly the same version that travis uses).  And ran the `nosetests`  like travis did. 
```
[~/repo/pubs][⎇ ][●][17:41]
[$] python --version
Python 3.5.0

[~/repo/pubs][⎇ ][●][17:41]
[$] nosetests
........................................................................................
----------------------------------------------------------------------
Ran 88 tests in 1.689s

OK
```
So there are some options:
1: I could fix the error that travis gives "blindly" and hope it works (because it works on my box either way). 
2: You could check out and run the tests (maybe you can confirm travis).
3: You have another idea... (which I'm sure will be better then mine)....

**TL;DR** *unable to reproduce*, what to do now?

**Comment by humm** [comment]
I see. I am going to look if I can reproduce it on my machine tomorrow. I'll keep you updated.

**Comment by omangin** [comment]
OK. It seems that the `version` argument for `ArgumentParser` was dropped. It seems that it was originally for compatibility with *optparse*.

You have to replace it with an explicit call to  `add_argument`.

See last line from <https://docs.python.org/2.7/library/argparse.html#upgrading-optparse-code>.

**Comment by 73** [comment]
@omangin thx for having a look. I found that already. But if I am not able to reproduce the error travis is throwing, I have no reliable way to check whether or not it is the only error.

**Comment by humm** [comment]
Ok, I pulled your commit, rebased on fixes introduced by 0bfe921a, and added commit to make it python 3 compliant. You can see your work in the feat/new_config branch, soon in master. Thanks for all your work. I'll probably build on your changes to catch 'init' with the conf_parser.

**Comment by humm** [comment]
@73 
I want to add your name to the list of contributors. Should I add it as 73 or should I use your real name ?

**Comment by 73** [comment]
You can use my real one: Arnold Sykosch
